core/src/tests/testing_common.h

```diff
@@ -19,19 +19,18 @@
    02110-1301, USA.
 */
 
-#ifndef TESTING_COMMON_H
-#define TESTING_COMMON_H
+#ifndef BAREOS_TESTS_TESTING_COMMON_H_
+#define BAREOS_TESTS_TESTING_COMMON_H_
 
 #if defined(HAVE_MINGW)
 #  include "include/bareos.h"
 #endif
 
 #include "gtest/gtest.h"
-#include "gmock/gmock.h"
```

Comment:
Why removing gmock.h ? Wouldn't that failed usage for example of database mocking  ?

Reply:
Will be added only if needed (not the case actually)